components/Cards/src/MDCCard.h

```diff
@@ -45,6 +46,12 @@
  */
 @property(nonatomic, getter=isInteractable) IBInspectable BOOL interactable;
 
+/**
+ This is used to integrate the Ripple Beta component into the card collection cell.
+ Warning: Please do not conform to this as this is used to integrate the Beta Ripple component.
+ */
+@property(nonatomic, weak, nullable) id<MDCCardRippleDelegate> rippleDelegate;
```

Comment:
Consider moving this property and its protocol to a separate header file so that it’s not advertised as part of the component’s public api by default.

Reply:
Done.